features/step_definitions/installer.rb

```diff
@@ -0,0 +1,15 @@
+## installer related steps such as upgrade
+#
+Given /^I upgrade my cluster to:$/ do | table |
+  ensure_admin_tagged
+  _admin = admin
+  upgrade_timeout = 2 * 60 * 60  # 2 hrs.
+  opts = opts_array_to_hash(table.raw)
+  opts[:force] = true
+  target_version = opts[:to_image]
```

Comment:
@pruan-rht Thanks for creating this rb. I am wondering how 'to_image' opt will get its value? The nightly releases will keep getting pruned and we can't define a static value for 'to_image' in scenarios. Thinking...

Reply:
@anuragthehatter it's getting it from environment variable `UPGRADE_TARGET_VERSION` which is set by the user...perhaps I should clarify that in the comment section.

Reply:
Gotha. Sure, commnt would be helpful. Thanks!